engine/events/pokecenter_pc.asm

```diff
@@ -104,27 +104,26 @@ PC_CheckPartyForPokemon:
 	ret nz
 	ld de, SFX_CHOOSE_PC_OPTION
 	call PlaySFX
-	ld hl, .MustHavePokemonToUse
+	ld hl, .PokePCCantUseText
```

Comment:
"Pokecenter", not "Poke". (Likewise for this whole file.)

Reply:
I hope all of the `PlayersPC` labels are fine.